llama_stack/core/server/auth_providers.py

```diff
@@ -23,7 +23,7 @@
 )
 from llama_stack.log import get_logger
 
-logger = get_logger(name=__name__, category="auth")
+logger = get_logger(name=__name__, category="core")
```

Comment:
what is the rule to extract the category ? First path element ("core") or shouldn't it be last path element ("server") ?